.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -0,0 +1,59 @@
+name: Bug Report
+description: Report a bug with the project.
+type: "Bug"
+labels: ["needs triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        > [!IMPORTANT]
+        > Before filing a report, please ensure you are running a supported version of the project.
+  - type: textarea
+    attributes:
+      label: Project Version
+      description: |
+        Please include the version of the project you are running.
+        Additionally, include additional information around the dependencies you are using.

```

```suggestion
        Additionally, include additional information about the dependencies you are using.
```
is this intentional? I've never seen it used like this